src/controller/org.controller/org.controller.js

```diff
@@ -38,32 +38,44 @@ async function getCnas (req, res, next) {
 // Get the details of a single cna for the specified shortname
 async function getCna (req, res, next) {
   try {
+    const cnaShortName = req.ctx.org
```

Comment:
Change use of CNA to org for variables or organization for human readable messages